package.json

```diff
@@ -25,11 +25,10 @@
     "lib": "lib"
   },
   "dependencies": {
-    "appium-base-driver": "^2.0.1",
+    "babel-runtime": "=5.8.24",
+    "appium-base-driver": "=2.3.0",
```

Comment:
Reason for that was the e2e tests fail on later base-driver versions and i fixed it to the one that last passed on Travis. I had a quick look at the test and couldn't figure out what is different appium-base-driver to make the fake-driver test fail.

I will change this and you can see the failure on Travis if you have any quick solutions for it.

What is your preferred method here? Ignore the test or fix the version to last known passing?